connector/src/main/resources/org/jboss/as/connector/subsystems/resourceadapters/LocalDescriptions.properties

```diff
@@ -77,7 +77,7 @@ connection-definitions.dump-queued-threads-in-pool=Dump queued threads in the po
 connection-definitions.flush-idle-connection-in-pool=Flushes all idle connections in the pool.
 connection-definitions.flush-invalid-connection-in-pool=Flushes all invalid connections in the pool.
 connection-definitions.flush-gracefully-connection-in-pool=Flushes all connections gracefully in the pool.
-connection-definitions.flush-strategy=Specifies how the pool should be flushed in case of an error. Valid values are: FailingConnectionOnly, IdleConnections, and EntirePool.
+connection-definitions.flush-strategy=Specifies how the pool should be flushed in case of an error. Valid values are: FailingConnectionOnly (default), InvalidIdleConnections, IdleConnections, Gracefully, EntirePool, AllInvalidIdleConnections, AllIdleConnections, AllGracefully, AllConnections
```

Comment:
As told there it's already done, because DS and RA share that attribute (CommonPool)